README.md

```diff
@@ -53,3 +53,7 @@ Software converts simple variants (SNVs, Indels), along with zygosity and phase
 ### License and Limitations
 
 Software is available for use under an [Apache 2.0 license](https://opensource.org/licenses/Apache-2.0), and is intended solely for experimental use, to help further Genomics-EHR integration exploration. Software is expressly not ready to be used with identifiable patient data or in delivering care to patients. Code issues should be tracked here. Comments and questions can also be directed to info@elimu.io or srikarchamala@gmail.com.
+
```

Comment:
@rhdolin do you think we need a section like this ? I feel the link to manual page in Introduction "Details of the translation logic are on the [manual page](docs/Manual.md)." is enough.